docs/ontap-metrics.md

```diff
@@ -11168,42 +11171,38 @@ Number of disk write operations initiated each second for storing data or metada
 
 ### rw_ctx_cifs_giveups
 
-Array of number of give-ups of CIFS ops because they rewind more than a certain threshold, categorized by their rewind reasons.
+Array of number of given-ups of cifs ops because they rewind more than a certain threshold, categorized by their rewind reasons.
 
 | API    | Endpoint | Metric | Template |
 |--------|----------|--------|---------|
-| REST | `api/cluster/counter/tables/rewind_context` | `cifs_give_ups`<br><span class="key">Unit:</span> none<br><span class="key">Type:</span> delta<br><span class="key">Base:</span>  | conf/restperf/9.16.0/rwctx.yaml |
-| ZAPI | `perf-object-get-instances rw_ctx` | `cifs_giveups`<br><span class="key">Unit:</span> none<br><span class="key">Type:</span> delta<br><span class="key">Base:</span>  | conf/zapiperf/cdot/9.8.0/rwctx.yaml |
```

Comment:
This need to be restore as-is, our clusters are 9.15 and these counters are from 9.16.